Additional notes for destroy(). (#1802)
cocos-creator/creator-docs: en/scripting/create-destroy.md

### destroy and removeFromParent

> After calling `removeFromParent` of a node, in-memory data is not released because there is some logical problem that causes the program to still refer to the object. **So if a node is no longer used, please call its `destroy` directly instead of `removeFromParent`, otherwise a memory leak will result.**

Reviewer: ```suggestion
After calling `removeFromParent` on a node, in-memory data is not released because there is some logical problem that causes the program to still refer to the object. **If a node is no longer used, please call `destroy` directly instead of `removeFromParent`, otherwise a memory leak will result.**
```

> After calling `removeFromParent` of a node, in-memory data is not released because there is some logical problem that causes the program to still refer to the object. **So if a node is no longer used, please call its `destroy` directly instead of `removeFromParent`, otherwise a memory leak will result.**

Author: ```suggestion
After calling `removeFromParent` interface on a node, the node is not released from memory because the engine still holds its data internally. **If a node is no longer used, please call its `destroy` directly instead of `removeFromParent`, otherwise, a memory leak will result.**
```
